plugins/timelib/timelib.hpp

```diff
@@ -52,7 +52,7 @@ unsigned int tlExtractTimeFromTimeStruct(const struct tm* timeInfo);
 
 void tlLocalTime_r(const time_t* clock, struct tm* timeInfoPtr);
 void tlGMTime_r(const time_t* clock, struct tm* timeInfoPtr);
-time_t tlMKTime(struct tm* timeInfoPtr, bool inLocalTimeZone = true);
+time_t tlMKTime(struct tm* timeInfoPtr, bool inLocalTimeZone = false);
```

Comment:
trivial: This could be a static function - it isn't used outside the c++ file.

Reply:
I agree.  That changes turned out to be much less than I originally thought.  I think the impact is very minor, and requires zero coding changes (or even awareness) on the ECL coder's part.

Reply:
Actually, I believe that all three of those functions are used only within this module.  They are available for everyone, but no one has taken advantage.  They could be removed from the header entirely.